package/src/lib/components/tour-overlay/TourOverlay.component.tsx

```diff
@@ -178,6 +181,17 @@ export const TourOverlay = forwardRef<TourOverlayRef, TourOverlayProps>((props,
               />
             </Mask>
           </Defs>
+          {spotOverlayProps &&
+            <ShapeMask
+              spot={spot}
+              setReference={refs.setReference}
```

Comment:
Not 100% sure on the use case of `setReference`, maybe could be removed for this overlay